crates/iota-indexer/src/processors/network_metrics_processor.rs

```diff
@@ -25,21 +27,29 @@ where
     S: IndexerAnalyticalStore + Clone + Sync + Send + 'static,
 {
     pub fn new(store: S, metrics: IndexerMetrics) -> NetworkMetricsProcessor<S> {
-        let network_processor_metrics_batch_size =
-            std::env::var("NETWORK_PROCESSOR_METRICS_BATCH_SIZE")
```

Comment:
Doesn't this break backward compatibility? If someone is configuring the service already using this env variable they will not get the expected results.

Reply:
Yes, it does. However we didn't advise anyone to make use of the existing `NETWORK_PROCESSOR_METRICS_BATCH_SIZE` env variable as far as I know. Also we don't use it in any prod deployment (also checked the docker-compose). And since the network metrics processor logic changed and behaves differently now, I guess even if it breaks it is okay to run with the default settings.